samples/todo-app/deployment-arm.yaml

```diff
@@ -0,0 +1,224 @@
+apiVersion: apps/v1
```

Comment:
It looks as if the `deployment-arm.yaml` and `deployment.yaml` manifests differ only in the image. Would it be possible to publish a docker manifest to the container registry that would allow us to use the same image name to refer to the images for every OS and architecture, so we'd only need one deployment file?

Reply:
updated this. Thank you @peterbom.